lib/profiling/README.md

```diff
@@ -175,3 +175,16 @@ Example output (from nvprof):
           API calls:   59.10%  122.02us     34  3.5880us  2.4540us  14.057us  cuMemcpyHtoDAsync
                        34.73%  71.711us     10  7.1710us  4.6830us  25.625us  cuLaunchKernel
                         6.17%  12.729us      3  4.2430us  2.3700us  7.7330us  cuMemsetD32Async
+
+### LFRic Timer
```

Comment:
Please add 'lfric' to the list of possible TARGETs at the top of this file.

Reply:
Indeed.